app/kubernetes/components/yaml-inspector/yamlInspector.html

```diff
@@ -6,6 +6,7 @@
     placeholder="# Define or paste the content of your manifest here"
     read-only="true"
     hide-title="true"
+    on-change="($ctrl.onChange)"
```

Comment:
is this change required for the tooltip ticket?

Reply:
removed